src/bin/pgcopydb/ld_test_decoding.c

```diff
@@ -715,6 +715,17 @@ parseNextColumn(TestDecodingColumns *cols,
 	cols->colnameStart = ptr;
 	cols->colnameLen = typA - ptr;
 
+	/*
+	 * The test_decoding plugin already quotes keywords. We remove the quotes
+	 * here because we will add them later during the transformation to SQL.
+	 */
+	if (cols->colnameStart[0] == '"')
+	{
+		cols->colnameStart++;
+		cols->colnameLen -= 2;
+	}
+
+
```

Comment:
This looks strange to me. I understand it's the smallest amount of code changes required to pass the tests, but we're now removing quotes only to add them again later. What about instead we only decide to add quotes in `stream_write_...` functions when it's not already quoted?

Reply:
That's how I started with :) But yeah, I need to handle this in `stream_write_{insert, delete, update},` functions. I could do that.